CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## unreleased
 * BraintreeCore
   * Fix analytics bug where sessionID value in analytics payload was inaccurate; send separate FPTI POST requests per unique sessionID
+* Breaking Changes
```

Comment:
Not sure if it'd be easier to create a v7 section of the changelog for now. May make it easier to reason about what is a v7 vs v6 change with merge conflicts too.

Reply:
Good call, updated!